zipkin/binding/pyramid/pyramidhook.py

```diff
@@ -73,3 +88,19 @@ def log_response(request):
         local().reset()
 
     return log_response
+
+
+class tween_factory(object):
+    def __init__(self, handler, registry):
+        self.handler = handler
+        self.registry = registry
+
+    def __call__(self, request):
+        class FakeEvent(object):
```

Comment:
This is not a FakeEvent actually.

Could be a named ZipkinTweenEvent or something like this.